### DISCUSSION TITLE
Linux can’t find game instance!

### DISCUSSION THREAD

**Discussion by Mattchu7211** [whs/runekit#28] [Q&A Linux]
Hello I’m running this on Linux with the steam client. And it can’t find the game instance, any tips?

**Comment by pekempy** [comment]
@Mattchu7211 this repository is unmaintained, I recommend using alt1-electron, the official build. This works on Linux (I use Arch [btw])
https://github.com/skillbert/alt1-electron

**Reply by schoentoon** [reply]
Alt1 electron is still quite early stage (and doesn't seem very maintained either, last commit in August just saying). Plus depending on what OP needs it may not work either way, so maybe not the best idea to redirect someone there right away.

**Reply by pekempy** [reply]
It's more maintained than 13 months ago for this repo- and I've had no issues with it so far including adding external apps

**Comment by schoentoon** [comment]
It likely won't recognize it as it can't find the process due to the wine/proton layer in between (last I checked Jagex never uploaded the Linux client to Steam at least). Is there any reason you are unable to run the client natively, perhaps even through Flatpak or alike?
